x-pack/platform/plugins/shared/observability_ai_assistant/server/service/client/index.ts

```diff
@@ -100,7 +102,57 @@ export class ObservabilityAIAssistantClient {
       scopes: AssistantScope[];
     }
   ) {}
+  async inferNER(chunks: InferenceChunk[]) {
+    const promises = chunks.map(async ({ chunkText, charStartOffset }) => {
+      const response = await this.dependencies.esClient.asCurrentUser.ml.inferTrainedModel({
+        model_id: 'elastic__distilbert-base-uncased-finetuned-conll03-english',
+        docs: [
```

Comment:
is the token limit per doc or per inference call?

Reply:
Per inference call.  The API truncates anything larger and you'd get back is_truncated: true and ignores anything larger.